### PR TITLE
Equivalence to telescope

### PR DESCRIPTION
**Pull request by AdamDM**
Changes include:
 - better naming for equivalence tasks
 - Id's are now sent properly

### CODE REVIEW

File: src/main/java/org/atlasapi/equiv/EquivalenceUpdatingWorker.java [hunk 1]
```diff
@@ -33,6 +38,7 @@
     
     private final SubstitutionTableNumberCodec idCodec
         = SubstitutionTableNumberCodec.lowerCaseOnly();
+    private final OwlTelescopeReporter telescopeProxy;
```
Comment (alianos-) [inline src/main/java/org/atlasapi/equiv/EquivalenceUpdatingWorker.java]: You can refactor this to "telescope" as its not a proxy to another class anymore.

File: src/main/java/org/atlasapi/equiv/channel/updaters/SourceSpecificChannelEquivalenceUpdater.java [hunk 2]
```diff
@@ -69,6 +75,12 @@ private void setAndUpdateEquivalents(Channel candidate, Channel subject) {
             channelWriter.createOrUpdate(subject);
         }
 
+        telescopeProxy.reportSuccessfulEvent(
+                subject.getId(),
```
Comment (TjlHope) [inline src/main/java/org/atlasapi/equiv/channel/updaters/SourceSpecificChannelEquivalenceUpdater.java]: Is there no failed case?

File: src/main/java/org/atlasapi/reporting/telescope/OwlTelescopeReporters.java [hunk 3]
```diff
@@ -7,10 +7,29 @@
...
     BBC_NITRO_INGEST("bbc-nitro-ingester", "BBC Nitro Ingester"),
-    BBC_NITRO_INGEST_OFFSCHEDULE("bbc-nitro-ingester-offschedule", "BBC Nitro off-schedule Ingester"),
+    BBC_NITRO_INGEST_OFFSCHEDULE(
+            "bbc-nitro-ingester-offschedule",
+            "BBC Nitro off-schedule Ingester"
+    ),
     BBC_NITRO_INGEST_PICKS("bbc-nitro-ingester-picks","Mbst Picks Content Group Updater"),
     BBC_NITRO_INGEST_API("bbc-nitro-ingester-api", "BBC Nitro API Ingester"),
     BBC_NITRO_INGEST_CHANNELS("bbc-nitro-ingester-channels", "BBC Nitro Channel Ingester"),
+    MOCK_REPORTER("mock-reporter", "Mock reporter"), //used by OwlTelescopeProxyMock
```
Comment (TjlHope) [inline src/main/java/org/atlasapi/reporting/telescope/OwlTelescopeReporters.java]: Not so keen on having a test one in the main code - separate test enum?

File: src/main/java/org/atlasapi/equiv/EquivalenceUpdatingWorker.java [hunk 4]
```diff
@@ -96,5 +96,18 @@ private Content resolveUri(String uri) {
...
-    
+
+    private void rotateTelescope() {
+        if (telescope == null) {
+            telescope = OwlTelescopeReporter.create(
+                    OwlTelescopeReporters.EQUIVALENCE_UPDATING_WORKER,
+                    Event.Type.EQUIVALENCE
+            );
+            telescope.startReporting();
+            timeSinceTelescopeRotated = DateTime.now();
```
Comment (TjlHope) [inline src/main/java/org/atlasapi/equiv/EquivalenceUpdatingWorker.java]: I'd personally do a 'timeToNextRotate' method, then you don't need the +1 each time, but that's just me nitpicking

File: src/main/java/org/atlasapi/equiv/EquivalenceUpdatingWorker.java [hunk 5]
```diff
@@ -96,5 +96,18 @@ private Content resolveUri(String uri) {
...
+        } else if (timeSinceTelescopeRotated.plusHours(1).isBeforeNow()) {
+            telescope.endReporting();
+            telescope.startReporting();
```
Comment (TjlHope) [inline src/main/java/org/atlasapi/equiv/EquivalenceUpdatingWorker.java]: You need a new telescope, once you've done `end` you can't `start` again

File: src/main/java/org/atlasapi/equiv/channel/updaters/SourceSpecificChannelEquivalenceUpdater.java [hunk 6]
```diff
@@ -69,6 +75,12 @@ private void setAndUpdateEquivalents(Channel candidate, Channel subject) {
...
+                subject.getAliases(),
+                subject
+        );
+
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/equiv/channel/updaters/SourceSpecificChannelEquivalenceUpdater.java]: the id is already there, but I should see if I can check if createOrUpdate fails anyway

File: src/main/java/org/atlasapi/reporting/telescope/OwlTelescopeReporters.java (hunk 3 above)
Comment (AdamDM) [inline src/main/java/org/atlasapi/reporting/telescope/OwlTelescopeReporters.java]: not sure what this is, it came over from Andreas's code when I had merge conflicts

File: src/main/java/org/atlasapi/reporting/telescope/OwlTelescopeReporters.java [hunk 7]
```diff
@@ -7,10 +7,29 @@
...
+    EQUIVALENCE("equivalence", "Equivalence"),
+    EQUIVALENCE_UPDATING_WORKER("equivalence-updating-worker", "Equivalence Updating Worker"),
+    CONTENT_EQUIVALENCE_UPDATE_TASK(
+            "content-equivalence-update-task",
+            "Content Equivalence Update Task"
+    ),
+    MANUAL_EQUIVALENCE("manual-equivalence", "Manual Equivalence"),
+    MANUAL_CHANNEL_EQUIVALENCE("manual-channel-equivalence", "Manual Channel Equivalence"),
+    QUERY_EXECUTOR_EQUIVALENCE("query-executor-equivalence", "Query Executor Equivalence")
     ;
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/reporting/telescope/OwlTelescopeReporters.java]: code style says <100 line length though

**Pull request closed without merging**